lambda_src/drivers/process_boto3.py

```diff
@@ -3,21 +3,53 @@
 import boto3
 from botocore.response import StreamingBody
 
+from utils import pick
```

Comment:
This should be a relative import.

Reply:
good catch -- was testing without top level package